File: LibraryMe.UI/LibraryMe/src/app/Features/Borrowings/models/borrowing-dto.ts
import {BookShortcutDTO} from "../../Books/models/book-shortcut-dto";

export interface BorrowingDTO {
    dateCreated: string;
    dueDate: string;
    borrower: string;
    borrowerVisitorCardId: number;
    status: string;
    books: BookShortcutDTO[];
}

File: LibraryMe.API/BookLibrary.DAL/Models/DTO/BorrowingDTO.cs
namespace BookLibrary.DAL.Models.DTO
{
    public class BorrowingDTO
    {
        public DateTime DateCreated { get; set; }
        public DateTime DueDate { get; set; }
        public string Borrower {  get; set; }
        public int BorrowerVisitorCardId { get; set; }
        public string Status { get; set; }
        public IEnumerable<BookShortcutDTO> Books { get; set; }
    }
}
